aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -183,7 +183,7 @@ static void abs_kernel(TensorIteratorBase& iter) {
 }
 
 static void angle_kernel(TensorIteratorBase& iter) {
-  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES_AND2(kBFloat16, kHalf, iter.common_dtype(), "angle_cpu", [&]() {
+  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES_AND3(kComplexHalf, kBFloat16, kHalf, iter.common_dtype(), "angle_cpu", [&]() {
```

Comment:
Let's not add half or complex half functions to the cpu, it's very slow, half type is not meant to be used on the cpu.

Reply:
Oops, okay! will remove it.

Reply:
@ngimel since `half` was already supported, wouldn't removing it be BC breaking?

Reply:
> @ngimel since half was already supported, wouldn't removing it be BC breaking?

Make sense to me too.